README.md

```diff
@@ -270,19 +269,19 @@ The `.tpd` files are processed using the [TPD Editor](https://github.com/eclipse
 which resolves the specified dependencies and creates a `.target`.
 The process is:
 
-  1. Install the TPD Editor, if necessary
-     - Use _Help > Install New Software_ and specify `http://download.eclipse.org/cbi/tpd/3.0.0-SNAPSHOT/`
-       as the location.
-     - Restart Eclipse when prompted
-  2. Open the `.tpd` file in Eclipse.
-  3. Make any necessary changes and save.
-     - Note that the TPDs specify artifacts using their _p2 identifiers_.
-       Bundles are specified using their OSGi Bundle Symbolic Name (e.g.,
-       `org.eclipse.core.runtime`).
-       Features are specified using their Feature ID suffixed with `.feature.group`
-       (e.g., `org.eclipse.rcp.feature.group`).
-  4. Select the file in the Package Explorer, right-click, and choose _Create Target Definition File_
-     to update the corresponding .target file.
+   1. Install the TPD Editor, if necessary
```

Comment:
I know markdown allows making every number in a list 1, but it is easier to read in source code if we don't do that.

Reply:
That's a good point. Updated with explicit numbers.